src/entities/container-box/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { ContainerBoxStyled } from '@/entities/container-box/style.ts'
+
+type Props = {
+  children: React.ReactNode
+}
+
```

Comment:
rewrite it like this:
export const ContainerBox: FC<PropsWithChildren> = ({children})